packages/next/src/sdk/client.tsx

```diff
@@ -15,20 +25,24 @@ import { CreateSdkContextReturn, SdkProviderProps } from "./types";
  * import { createSdkContext } from "@vue-storefront/next/client";
  * import { getSdk } from "../../sdk.config.ts";
  *
- * export const [SdkProvider, useSdk] = createSdkContext(getSdk());
+ * export const [AlokaiProvider, useSdk] = createSdkContext(getSdk());
  * ```
- * Then use the `SdkProvider` in the root component of your application.
+ * Then use the `AlokaiProvider` in the root component of your application.
  * For Pages Router it would be the `pages/_app.tsx` file,
  * and for the App Router it would be the `app/layout.tsx` file.
- * Finally you can use the `useSdk` in any client component of your application.
+ * Finally, you can use the `useSdk` in any client component of your application.
  * @returns [SdkProvider, useSdk] - The SDK provider and the `useSdk` hook.
  */
 export function createSdkContext<
```

Comment:
This function's name could be changed to `createAlokaiContext` yet it will be a breaking change.

Reply:
Sooner or later we will need to do that, so maybe at this point when we making a lot of changes in Next.js ;)

Reply:
Yeah, i've already changed that ;)